rules/application/kubernetes/audit/kubernetes_audit_privileged_pod_creation.yml

```diff
@@ -0,0 +1,29 @@
+title: Privileged Container Deployed
+id: c5cd1b20-36bb-488d-8c05-486be3d0cb97
+status: experimental
+description: |
+    Detects the creation of a "privileged" container, an action which could be indicative of a threat actor mounting a container breakout attacks.
+    A privileged container is a container that can access the host with all of the root capabilities of the host machine. This allows it to view, interact and modify processes, network operations, IPC calls, the file system, mount points, SELinux configurations etc. as the root user on the host.
+    Various versions of "privileged" containers can be specified, e.g. by setting the securityContext.privileged flag in the resource specification, setting non-standard Linux capabilities, or configuring the hostNetwork/hostPID fields
+references:
+    - https://microsoft.github.io/Threat-Matrix-for-Kubernetes/techniques/Privileged%20container/
+    - https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_finding-types-kubernetes.html#privilegeescalation-kubernetes-privilegedcontainer
+    - https://www.elastic.co/guide/en/security/current/kubernetes-pod-created-with-hostnetwork.html
+    - https://www.elastic.co/guide/en/security/current/kubernetes-container-created-with-excessive-linux-capabilities.html
+author: Leo Tsaousis (@laripping)
+date: 2024/03/08
+tags:
+    - attack.t1611
+logsource:
+    category: application
+    product: kubernetes
+    service: audit
+detection:
+    selection:
+        verb: create
+        resource: pods
+        capabilities: "*"
```

Comment:
Same question as for hostpath. Is this saying any field or the existence of the field?

Reply:
Same as above. Presence of this Sigma filter will force-include the respective ELK filter, matching only events that have this present (with any value)